packages/vaadin-themable-mixin/vaadin-themable-mixin.d.ts

```diff
@@ -1,3 +1,4 @@
+import { CSSResultGroup } from 'lit';
```

Comment:
Please add license comment to this file.

Reply:
Done.